pom.xml

```diff
@@ -42,7 +42,7 @@
   </organization>
 
   <properties>
-    <hadoop.version>2.7.3</hadoop.version>
+    <hadoop.version>2.8.0</hadoop.version>
```

Comment:
Yes it was mainly for LDAP support. I can write my custom class for the same rather than using Hadoop-auth LdapAuthenticationHandler.